analyzers/src/SonarAnalyzer.Common/Helpers/SymbolHelper.cs

```diff
@@ -22,12 +22,15 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
+using System.Reflection;
 using Microsoft.CodeAnalysis;
 
 namespace SonarAnalyzer.Helpers
 {
     internal static class SymbolHelper
     {
+        private static readonly PropertyInfo IsRecordPropertyInfo = typeof(ITypeSymbol).GetProperty("IsRecord");
```

Comment:
The `PropertyInfo` part is visible from the type. It's more relevant what type it relates to.
```suggestion
        private static readonly PropertyInfo ITypeSymbolIsRecord = typeof(ITypeSymbol).GetProperty("IsRecord");
```